README.md

```diff
@@ -1,88 +1,130 @@
 
-> NOTE: This application uses Twilio's new Programmable Video Rooms API, which is currently available in Beta. The docs are [here](https://media.twiliocdn.com/sdk/ios/video/v1.0/docs/).
+[ ![Download](https://img.shields.io/badge/Download-iOS%20SDK-blue.svg) ](https://www.twilio.com/docs/api/video/download-video-sdks#ios-sdk)
+[![Docs](https://img.shields.io/badge/iOS%20Docs-OK-blue.svg)](https://media.twiliocdn.com/sdk/ios/video/releases/1.0.0-beta14/docs/index.html)
 
-## Up and Running
+# Twilio Video Quickstart for Swift
 
-To get started, we'll make a 1:1 video call between the iPhone simulator in XCode and your own iOS device.
+Get started with Video on iOS:
 
-1) Download this project and run `pod install` to install TwilioVideo.framework. Open VideoQuickStart.xcworkspace in Xcode.
+- [Setup](#setup) - Get setup
+- [Quickstart](#quickstart) - Run the quickstart app
+- [Examples](#examples) - Run the sample applications
+- [Setup an Access Token Server](#setup-an-access-token-server) - Setup an access token server
+- [More Documentation](#more-documentation) - More documentation related to the iOS Video SDK
+- [Issues & Support](#issues-and-support) - Filing issues and general support
 
-2) Create a Twilio Video [Configuration Profile](https://www.twilio.com/console/video/profiles) in the Twilio Console. If you haven't used Twilio before, welcome! You'll need to [Sign up for a Twilio account](https://www.twilio.com/try-twilio). It's free!
+## Setup 
 
-3) Generate an [Access Token](https://www.twilio.com/console/video/dev-tools/testing-tools) in the Twilio Console. Pick a name for your user identity (such as "Alice"). Leave this web page open, because you'll use it as the other side of the video chat.
+### Twilio Library Setup for the Project
 
-4) Copy and paste the access token into ViewController.swift.
+You will need to add the Twilio Video library to the project to compile and run. You can do this using the [Cocoapods](https://cocoapods.org/) dependency management system. 
 
-5) Build and run the app on a physical device (iPhone, iPod Touch, etc.)
+#### Using Cocoapods
 
-6) On the same web page where you generated the token, generate another Access Token for another identity (such as "Bob")
+First, you will need to have Cocoapods 1.0.0+ installed on your Mac, so go ahead and do that if you haven't already - the directions are here: [Getting Started with Cocoapods](https://guides.cocoapods.org/using/getting-started.html). If you're not sure, type `pod --version` into a command line.
+
+Next, just run `pod install` from the command line in the top level directory of this project. Cocoapods will install the Twilio library and then set up a .xcworkspace file that you will use to run your project from now on.
+
+**Note: If you've used [Twilio's Cocoapods Spec Repo](https://github.com/twilio/cocoapod-specs) before you may need to update it by running `pod repo update twilio` before installing dependencies.** 
```

Comment:
Use something like:

Note: You may need to update the CocoaPods [Master Spec Repo](https://github.com/CocoaPods/Specs) by running `pod repo update master` in order to fetch the latest specs of TwilioVideo.